Civi/Share/Message.php

```diff
@@ -135,27 +138,96 @@ public function send($local_node_id = null)
    *
    * @return void
    */
-  public function processOnNode($local_node_id)
+  public function processChanges($local_node_id)
```

Comment:
Is `$local_node_id` supposed to be the source node? Is this the internal or external ID of the node?